keyboards/gray_studio/space65/readme.md

```diff
@@ -1,11 +1,13 @@
 # Gray Studio Space65
 
-A 65% keyboard with RGB underglow, backlighting and USB C, whose design was inspired by the Voyager I space probe and Apple II home computer.
+A 65% keyboard with RGB underglow, backlighting, and USB C. Its design was inspired by the Voyager I space probe and Apple II home computer.
 
 Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
 Hardware Supported: Gray Studio Space65  
 Hardware Availability: [Geekhack Group Buy](https://geekhack.org/index.php?topic=98768.0)
 
+**Reset Sequence:** Using this firmware sets `BOOTLOADER_ENABLE` to `lite`. While plugging in, hold the top left key, commonly programmed as `Esc` to put your board into bootloader mode. 
```

Comment:
```suggestion
**Reset Sequence:** By default, `BOOTMAGIC_ENABLE` is set to `lite` on this board. Hold the top left key (commonly programmed as Esc) while plugging in to put the board into bootloader mode.
```

Reply:
I disagree with that statement. It has not been set to lite by default. I explicitly set it to lite.

Reply:
That's what the sentence says, no? Note the “on this board” part. Anyway, I changed it because I thought “Using this firmware” sounded weird. Makes it sound like using _QMK_ sets `BOOTMAGIC_ENABLE` to `lite` 🤷‍♀️ Dunno how you could phrase that differently without it sounding awkward.